aten/src/ATen/function_wrapper.py

```diff
@@ -147,6 +147,16 @@ def TypedDict(name, attrs, total=True):  # type: ignore
     return table->getOp<${return_type} (${formals_types})>(tensorTypeIdToBackend(type_id()), is_variable())(${method_actuals});
 }
 """)
+C10_TENSOR_METHOD_DEFINITION = CodeTemplate("""\
+inline ${return_type} Tensor::${api_name}(${method_formals})${const_mark} {
+    static c10::OperatorHandle op = c10::Dispatcher::singleton().findSchema({"aten::${name}", "${overload_name}"}).value();
+    if (is_variable()) {
```

Comment:
Is it possible to move the variable logic inside the dispatch call?

Reply:
yes, planning to do this in a future PR. This is the easier change for now.

Reply:
These will have to be collapsed into one call post Variable lives in TensorTypeSet in any case